articles/active-directory/devices/faq.md

```diff
@@ -301,6 +301,11 @@ If a password is changed outside the corporate network (for example, by using Az
 - For iOS and Android, you can use the Microsoft Authenticator application **Settings** > **Device Registration** and select **Unregister device**.
 - For macOS, you can use the Microsoft Intune Company Portal application to unenroll the device from management and remove any registration. 
 
+For Windows 10 devices, if needed, this process can be automated with the WPJ removal tool that can be downloaded here: [WPJCleanUp.zip](https://download.microsoft.com/download/8/e/f/8ef13ae0-6aa8-48a2-8697-5b1711134730/WPJCleanUp.zip)
```

Comment:
Define your acronym "WPJ"
"here" and "WPJCleanUp.zip" are bad link text. For more information see the Docs contributor guide and the section [How to write good link text](https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#how-to-write-good-link-text).